index.html

```diff
@@ -3,9 +3,9 @@
   <head>
     <meta charset="utf-8">
     <title>ReactJS.berlin &mdash; Berlin's ReactJS Usergroup</title>
-    <meta name="description" content="ReactJS.berlin is a usergroup focused on ReactJS and related
-     topics. We meet regularly each month at 7:00 p.m. @bitcrowd office,
-     Oranienstr. 6 in Berlin-Kreuzberg.">
+    <meta name="description"
+      content="ReactJS.berlin is was an early usergroup focused on ReactJS and related
```

Comment:
I think this comment is outdated now, because this looks right now:

![Screenshot 2019-06-12 at 16 42 52](https://user-images.githubusercontent.com/23569001/59360934-4d38e900-8d31-11e9-926c-3dac2fa35b60.png)